OpenMarket/OpenMarket/Controller/ViewController.swift

```diff
@@ -0,0 +1,139 @@
+import UIKit
+
+@available(iOS 14.0, *)
+class ViewController: UIViewController {
+    private enum Section: CaseIterable {
+        case main
+    }
+    // MARK: - Properties
+    @IBOutlet weak var segment: SegmentedControl!
+    @IBOutlet weak var listCollectionView: UICollectionView!
+    @IBOutlet weak var gridCollectionView: UICollectionView!
+    
+    private var productList = [ProductInformation](){
```

Comment:
공백이 없네요~ 실수인 것 같아요! 😉
```suggestion
    private var productList = [ProductInformation]() {
```

Reply:
수정 완료했습니다! 😊